### DISCUSSION TITLE
Struct e classe

### DISCUSSION THREAD

**Discussion by tadeu-braga** [balta-io/forum#114] [Tira Dúvidas] [answered]
Qual diferença e entre Struct e classe? pra mim elas fazem a utilidade. porém vejo alguma programadores usando-as juntos.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @tadeu-braga, como vai? A classe é um tipo de referência e pode ter toda estrutura da orientação a objetos. A estrutura é apenas um agrupamento de dados, que resulta em um tipo de valor. Mais a frente tem o curso de OOP também.
